lib/jekyll/commands/serve/livereload_assets/livereload.js

```diff
@@ -0,0 +1,1183 @@
+(function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```

Comment:
😨  What is this?

Can we have a non-minified version of this somewhere? Or at least a comment with a link to where the original source can be found?

Reply:
FWIW you should be able to format it in you browser developer tools.

Reply:
I just mean that all the variable names have been replaced with single letters. Even formatted, this is not readable code.

Reply:
We can de-obfuscate in-browser? I did not know that..!!

Reply:
https://github.com/livereload/livereload-js/blob/master/dist/livereload.js

Reply:
Yes, that's the source of it.  Depending on how y'all like to do things, it might make sense to pull that in via whatever the JS dependency tool of the month is.